<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>photoTanDialog</class>
 <widget class="QWidget" name="photoTanDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Order confirmation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="10,0">
     <item row="1" column="0">
      <widget class="QGraphicsView" name="graphicsView"/>
     </item>
     <item row="0" column="1" rowspan="2">
      <widget class="QTextBrowser" name="infoText">
       <property name="styleSheet">
        <string notr="true">background:transparent</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="acceptRichText">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>To confirm this order enter the tan displayed by your &amp;generator</string>
       </property>
       <property name="buddy">
        <cstring>tanInput</cstring>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="tanInput"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tanInput</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>infoText</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
